lib/src/avatar_glow.dart

```diff
@@ -76,14 +80,27 @@ class _AvatarGlowState extends State<AvatarGlow>
 
     // Check if the widget is still mounted before starting the animation.
     if (mounted) {
-      if (widget.repeat) {
-        _controller.repeat();
+      if (widget.glowRepeatCount > 0) {
```

Comment:
For reasons of readability, I would write this in a single If branch

`
if (mounted && widget.glowRepeatCount > 0)
`